[K8s] Resource Server Helm chart rework to use single deployment for all verticles

karthickp432001 opened the pull request:
- Modified values.yaml to remove all the verticles except apiServer
- Modified aws and azure resource-values files to have only resources and limits of single verticle
- updated the helm chart

karun-singh commented on K8s-deployment/Charts/resource-server/templates/deployment.yaml:
```diff
@@ -1,51 +1,51 @@
 kind: Deployment
 metadata:
-  name: rs-async
+  name: rs-api-server
```
Change deployment name and labels to resource-server, to avoid confusion. Also matching whats there in HPA
```suggestion
  name: resource-server
```

karun-singh commented on K8s-deployment/Charts/resource-server/example-aws-resource-values.yaml:
```diff
@@ -20,160 +20,6 @@ apiServer:
     requests: 
```
Increase the resources since all verticles will run in single pod now.

karun-singh commented on K8s-deployment/Charts/resource-server/example-azure-resource-values.yaml:
```diff
@@ -20,160 +20,6 @@ apiServer:
     requests: 
```
Increase the resources since all verticles will run in single pod now.

karun-singh commented on K8s-deployment/Charts/resource-server/values.yaml:
```diff
@@ -342,2097 +342,6 @@ apiServer:
-  extraEnvVarsCM: "rs-env"
-  ## @param async.extraEnvVarsSecret Name of existing Secret containing extra env vars for async nodes
-  ##
-  extraEnvVarsSecret: "rs-s3-env"
```
**We still need to pass this for the async verticle to work, please add it in above resourceServer block.**

karthickp432001 commented:
As per the suggestion I have updated the resources in the resource values yaml file
added the "rs-s3-env" in the values.yaml
updated metadata name and label to resource-server
